app/views/request-psp-test-account/index.njk

```diff
@@ -75,14 +78,13 @@
         Request submitted for Stripe test account
       <h1>
       <p class="govuk-body">
-        Thank you for requesting a Stripe test account
+        Thank you for requesting a Stripe test account.
```

Comment:
Does the content doc suggest that this should now be the title rather than `Request submitted for Stripe test account`?

Reply:
yeah, slight changed in the text. Updated to latest